content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -620,6 +620,11 @@ spec:
 
 PersistentVolumes binds are exclusive, and since PersistentVolumeClaims are namespaced objects, mounting claims with "Many" modes (`ROX`, `RWX`) is only possible within one namespace.
 
+### PersistentVolumes typed HostPath
+
+Kubernetes supports hostPath for development and testing on a single-node cluster. A hostPath PersistentVolume uses a file or directory on the Node to emulate network-attached storage.
```

Comment:
> Kubernetes supports hostPath for development and testing on a single-node cluster.

I know this is a quote, but I feel that it's misleading. For example, SIG Cluster Lifecycle uses `hostPath` volume mounts for control plane components with `kubeadm`, even in multi-node, multi-zone clusters that have a replicated control plane.

`hostPath` volumes are tied to a particular node, and an understanding of static Pods and / or scheduling is necessary to use these in real clusters. I think they have their place though.

@tengqm do you agree that we should look to reword this sentence?

Reply:
@sftim According to suggestions from SIG storage, `hostPath` is not the right volume to use in production. The `local` volume is recommended as an alternative.

Reply:
@sftim @tengqm Do you happen to know what we can do to get someone to approve and merge this PR?

Reply:
Hi @kedark3 

I still feel that this advice is misleading as it stands: there are several valid reasons to use `hostPath` volumes on production clusters that run across multiple nodes.

Would you be willing to revise this to clarify when it is and isn't appropriate to use `hostpath` volume / PV / PVC?

Reply:
@sftim @tengqm I feel like this is an issue that needs to be addressed by someone more experienced. I am willing to remove that statement altogether for now if that means my PR gets merged. I am pretty sure I read it somewhere else in the docs and copied here as is. I really wanted to add example reference for hostPath, I didn't mean to make a recommendation. Thoughts ? :smile:

Reply:
:+1: for simplifying the change as a way to move this forward

Reply:
@sftim I removed that statement.